app/Cardano/Shell/Features/Logging.hs

```diff
@@ -94,13 +95,14 @@ loggingCardanoFeatureInit loggingConfig = do
     let initLogging :: CardanoEnvironment -> NoDependency -> CardanoConfiguration -> LoggingParameters -> IO LoggingLayer
         initLogging _ _ _ _ = do
             pure $ LoggingLayer
-                    { llBasicTrace  = Trace.natTrace liftIO baseTrace
-                    , llLogDebug    = Trace.logDebug
-                    , llLogInfo     = Trace.logInfo
-                    , llLogNotice   = Trace.logNotice
-                    , llLogWarning  = Trace.logWarning
-                    , llLogError    = Trace.logError
-                    , llAppendName  = Trace.appendName
+                    { llBasicTrace    = Trace.natTrace liftIO baseTrace
+                    , llLogDebug      = Trace.logDebug
+                    , llLogInfo       = Trace.logInfo
+                    , llLogNotice     = Trace.logNotice
+                    , llLogWarning    = Trace.logWarning
+                    , llLogError      = Trace.logError
+                    , llAppendName    = Trace.appendName
+                    , llConfiguration = lpConfiguration loggingConfig
```

Comment:
This looks kind of weird. 
Isn't it possible to add the `bracket` function here and export it?

Reply:
yes, I am experimenting with exporting everything that is needed in the target app and get rid of the direct import of the logging library. Types and functions should be (re-)exported through the feature in `node-shell`.